src/database/schema/ChatSchema.ts

```diff
@@ -325,7 +325,7 @@ class ChatSchema implements BaseDbSchema {
       rawJson,
       status,
       isContinuous: false,
-      type: 'regular',
+      type: data?.message?.message_type ?? data?.message?.type,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is similar to the ones on lines 214 and 240, adding an additional fallback for the `type` property. It's a good change if there are cases where the type could be under `message_type` instead of `type`.
```diff
-      type: 'regular',
+      type: data?.message?.message_type ?? data?.message?.type,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->